test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -846,6 +847,71 @@ void IndicateMissingValuesOnnxConversionTest()
             Done();
         }
 
+        private class TextData
+        {
+            public string Text { get; set; }
+        }
+
+        [Fact]
+        void NgramOnnxConnversionTest()
```

Comment:
>NgramOnnxConnversionTest [](start = 13, length = 24)

Can you please add tests for WordTokenizing and ValueToKeyMapping as well?

Reply:
The pipeline below already covers WordTokenizing, ValueToKeyMapping and NgramTransforms. Did you mean separate tests for each of the transforms?

---
In reply to: [333957174](https://github.com/dotnet/machinelearning/pull/4312#discussion_r333957174) [](ancestors = 333957174)